docs/LogGuide.md

```diff
@@ -2821,3 +2853,399 @@ Parsed Log Line Examples:
+
+<a name="line268"></a>
+
+### Line 268 (0x10C): Countdown
+
+This log line is emitted whenever a countdown is started.
+
+`result` is `00` if successful, and non-zero if the attempt to start a countdown failed
+(e.g., if a countdown is already in progress, or if combat has started).
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=Countdown&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+268|[timestamp]|[id]|[worldId]|[countdownTime]|[result]|[name]
+
+Parsed Log Line Structure:
+[timestamp] 268 10C:[id]:[worldId]:[countdownTime]:[result]:[name]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>268)\|(?<timestamp>[^|]*)\|(?<id>[^|]*)\|(?<worldId>[^|]*)\|(?<countdownTime>[^|]*)\|(?<result>[^|]*)\|(?<name>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 268 (?<type>10C):(?<id>[^:]*):(?<worldId>[^:]*):(?<countdownTime>[^:]*):(?<result>[^:]*):(?<name>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+268|2024-02-29T15:19:48.6250000-08:00|10FF0001|0036|13|00|Tini Poutini|0ab734bdbcb55902
+268|2024-02-29T15:34:16.4280000-08:00|10FF0002|0036|20|00|Potato Chippy|0ab734bdbcb55902
+
+Parsed Log Line Examples:
+[15:19:48.625] 268 10C:10FF0001:0036:13:00:Tini Poutini
+[15:34:16.428] 268 10C:10FF0002:0036:20:00:Potato Chippy
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=Countdown&lang=en-US) -->
+
+<a name="line269"></a>
+
+### Line 269 (0x10D): CountdownCancel
+
+This log line is emitted whenever a currently-running countdown is cancelled.
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=CountdownCancel&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+269|[timestamp]|[id]|[worldId]|[name]
+
+Parsed Log Line Structure:
+[timestamp] 269 10D:[id]:[worldId]:[name]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>269)\|(?<timestamp>[^|]*)\|(?<id>[^|]*)\|(?<worldId>[^|]*)\|(?<name>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 269 (?<type>10D):(?<id>[^:]*):(?<worldId>[^:]*):(?<name>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+269|2024-02-29T15:19:55.3490000-08:00|10FF0001|0036|Tini Poutini|e17efb9d120adea0
+269|2024-02-29T15:34:22.8940000-08:00|10FF0002|0036|Potato Chippy|e17efb9d120adea0
+
+Parsed Log Line Examples:
+[15:19:55.349] 269 10D:10FF0001:0036:Tini Poutini
+[15:34:22.894] 269 10D:10FF0002:0036:Potato Chippy
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=CountdownCancel&lang=en-US) -->
+
+<a name="line270"></a>
+
+### Line 270 (0x10E): ActorMove
+
+An `ActorMove` packet is sent to instruct the game client to move an actor to a new position
+(for example, in UWU, when Titan turns prior to jumping to indicate the direction of his jump).
+The FFXIV client interpolates the actor's movement between the current position and the new position.
+Currently, these log lines are emitted only for non-player actors (id >= 0x40000000).
```

Comment:
I think this is a bit misleading? I'd rephrase it something like `... to a new position whenever they have been moved. This can be used to detect rapid movement which would otherwise be lost, for example, in UWU, when...`